src/machine/usb/hid/hid.go

```diff
@@ -50,6 +50,8 @@ func handler() {
 	}
 }
 
+var DefaultSetupHandler = setupHandler
```

Comment:
It cannot be set by overriding DefaultSetHandler.
Perhaps hid.SetHandler needs to be changed as well.

Reply:
This would probably be a feature addition. So I did not think it was necessary to address it in this PR.